modules/misc/news.nix

```diff
@@ -462,6 +462,18 @@ in
           where --no-startup-id option is necessary.
         '';
       }
+      {
+        time = "2017-11-17T10:36:10+00:00";
+        condition = config.xsession.windowManager.i3.enable;
+        message = ''
+          i3 window Manager module has been extended with the following options:
```

Comment:
Nicer to start with a capital letter. Maybe "The i3 window manager module …"? Btw, the m in Manager seems randomly capitalized :-)